### PR TITLE
Update doc links

### PR DESCRIPTION
**Pull request by ashfire908**
Part of https://github.com/faunadb/sales-engineering/issues/348.

Does not make any code changes.

### CODE REVIEW

File: src/Client.js [hunk 1]
```diff
@@ -42,7 +42,7 @@ var Promise = require('es6-promise').Promise;
  *   HTTP scheme to use.
  * @param {?number} options.port
  *   Port of the FaunaDB server.
- * @param {?string} options.secret FaunaDB secret (see [Reference Documentation](https://faunadb.com/documentation/objects#keys))
+ * @param {?string} options.secret FaunaDB secret (see [Reference Documentation](https://fauna.com/documentation/objects#keys))
```
Comment (erickpintor) [inline src/Client.js]: I can't remember if we're also renaming `FaunaDB` to `Fauna`. Did you rename those on other drivers?

Comment (erickpintor) [inline src/Client.js]: I've checked other PRs. I don't think we are. That's fine, I guess.

Comment (ashfire908) [inline src/Client.js]: The database is still called FaunaDB. I'm only updating links to the website. Did I do more than that somewhere?

Comment (erickpintor) [inline src/Client.js]: No. I was the one that was confused. You're fine.

**Pull request merged**